src/celeritas/ext/GeantImporter.cc

```diff
@@ -192,6 +192,71 @@ struct MatPropGetter
     }
 };
 
+//---------------------------------------------------------------------------//
+//! Map particles defined in \c G4MaterialConstPropertyIndex .
+auto& optical_particles_map()
+{
+    static std::unordered_map<std::string, PDGNumber> map
```

Comment:
You forgot the const
```suggestion
    static const std::unordered_map<std::string, PDGNumber> map
```